IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -1049,6 +1049,11 @@ - (BOOL)clearWithContext:(id<MSIDRequestContext>)context
                    error:(NSError **)error
 
 {
+    // Clear in-memory cache
+    self.appStorageItem = [MSAIMSIDMacCredentialStorageItem new];
```

Comment:
I think you're using wrong classes here :)

Reply:
```objc
        self.appStorageItem = [MSIDMacCredentialStorageItem new];
        self.sharedStorageItem = [MSIDMacCredentialStorageItem new];
```
I'm duplicating the initialization logic on [L404](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/jbzdarkid/clear_in_memory/IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m#L404).

Reply:
We don't have a class called MSAIMSIDMacCredentialStorageItem (with MSA prefix) in this library :)

Reply:
Whoops, sorry.

Reply:
@oldalton Anything else I need to merge this PR?